servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -1054,12 +1054,7 @@ public DomainList getDomainList(ResourceContext ctx, Integer limit, String skip,
             String modifiedSince) {
 
         final String caller = "getdomainlist";
```

Comment:
I originally had the caller (method name in lowercase) placed in the context in ZMSResources and then I was able to have the same line for every method: "final String caller = ctx.getApiName();".
The reason I didn't change it eventually as I noticed that it is sometimes used in authority checks (as the operation in verifyAuthorizedServiceOperation calls). So maybe it is best to keep it explicit.
Let me know what you think.